Index: openscap/src/OVAL/probes/probe/icache.c
===================================================================
--- openscap.orig/src/OVAL/probes/probe/icache.c	2013-05-17 15:47:38.656274116 +0200
+++ openscap/src/OVAL/probes/probe/icache.c	2013-05-17 15:47:38.612274115 +0200
@@ -447,6 +447,7 @@
  */
 static int probe_cobj_memcheck(size_t item_cnt)
 {
+#if !(defined(__FreeBSD__) || defined(__FreeBSD_kernel__))
 	if (item_cnt > PROBE_RESULT_MEMCHECK_CTRESHOLD) {
 		struct proc_memusage mu_proc;
 		struct sys_memusage  mu_sys;
@@ -474,6 +475,7 @@
 			return (1);
 		}
 	}
+#endif
 
 	return (0);
 }
