Index: openscap/src/OVAL/probes/unix/linux/dpkginfo.c
===================================================================
--- openscap.orig/src/OVAL/probes/unix/linux/dpkginfo.c	2013-03-15 16:13:48.281739616 +0100
+++ openscap/src/OVAL/probes/unix/linux/dpkginfo.c	2013-03-15 18:35:28.729359953 +0100
@@ -54,6 +54,8 @@
 #include <seap.h>
 #include <probe-api.h>
 #include <alloc.h>
+#include <common/assume.h>
+#include "common/debug_priv.h"
 
 
 #include "dpkginfo-helper.h"
